src/Grids/conformal_cubed_sphere_panel.jl

```diff
@@ -211,32 +288,49 @@ function conformal_cubed_sphere_panel(architecture::AbstractArchitecture = CPU()
                                       η = (-1, 1),
                                       radius = R_Earth,
                                       halo = (1, 1, 1),
-                                      rotation = nothing)
+                                      rotation = nothing,
+                                      non_uniform_conformal_mapping = false,
+                                      spacing_type = "geometric",
+                                      use_provided_conformal_mapping = false,
```

Comment:
```suggestion
                                      use_provided_conformal_mapping = false,
```
you probably do not need this kwarg, you can just say that if `provided_conformal_mapping` is not nothing you use it.

Reply:
Great suggestion! Fixed in commit [ca48969](https://github.com/CliMA/Oceananigans.jl/pull/4295/commits/ca4896946cdffc80d5da0eef69e2ad85b6da670f).